src/lib.rs

```diff
@@ -31,7 +31,7 @@ pub static WILDCARD_VERSION_REQ: Lazy<VersionReq> =
   Lazy::new(|| VersionReq::parse_from_specifier("*").unwrap());
 
 #[derive(Error, Debug, Clone)]
-#[error("Invalid version. {source}")]
+#[error("Invalid version")]
```

Comment:
Removed `{source}` from a lot of these because it was doing:

```
<error: Failed reading lockfile at 'C:\Users\david\AppData\Local\Temp\deno-cli-test5VezLp\deno.lock'
<
<Caused by:
<    0: Failed deserializing. Lockfile may be corrupt.
<    1: Invalid package requirement '@denotest/basic@^=1.0.0'. Invalid version requirement. Invalid specifier version requirement. Unexpected character.
<         ^=1.0.0
character.
<         ^=1.0.0
<         ~
<    3: Invalid specifier version requirement. Unexpected character.
<         ^=1.0.0
<         ~
<    4: Unexpected character.
<         ^=1.0.0
<         ~
```

Reply:
+1, these errors were super verbose and not really helpful